tests/acceptance/staff_admin/doi-test.js

```diff
@@ -133,161 +138,146 @@ module('Acceptance | staff_admin | admin', function(hooks) {
     assert.dom('[data-test-description]').exists();
     assert.dom('[doi-subject]').exists();
     assert.dom('#doi-language').exists();
-    assert.dom('[doi-contributor]').exists();
+    // assert.dom('[doi-contributor]').exists();
```

Comment:
I guess this test has been commented because DOI in the test enviroment has not contributor data? Could be this reenable after modifying that DOI?

Reply:
The intention is to enable these test, but they need more work. For example making sure we don't create additional DOIs in every test run without cleaning up. We can for example create draft DOIs and delete them in another test.